version/version.go

```diff
@@ -6,9 +6,7 @@ const Fix = "2"
 
 var (
 	// The full version string
-	Version = "0.1.2"
-
-	// GitCommit is set with --ldflags "-X main.gitCommit=$(git rev-parse HEAD)"
```

Comment:
Don't delete the comments. I think it's against the rule of golint, so just keep them.

Reply:
Well,  we should  append golint to the travis.ci :)

Reply:
Yes, it's on the TODO list :).